hasura.planx.uk/migrations/1682351174497_alter_table_public_uniform_applications_add_column_xml/up.sql

```diff
@@ -0,0 +1,4 @@
+alter table "public"."uniform_applications" add column "xml" xml
```

Comment:
I'm not super familiar with the `xml` type in Postgres, but it seems a suitable choice? 

Docs: https://www.postgresql.org/docs/current/datatype-xml.html